tests/openvino/test_modeling.py

```diff
@@ -1932,13 +2002,50 @@ def test_compare_to_transformers(self, model_arch):
             self.assertIsInstance(getattr(ov_model, additional_part), MODEL_PARTS_CLS_MAPPING[additional_part])
         self.assertIsInstance(ov_model.config, PretrainedConfig)
         inputs = ov_model.preprocess_inputs(**preprocessors, text=prompt, image=self.IMAGE.resize((600, 600)))
-        # pytorch minicpmv and internvl are not designed to be used via forward
+        transformers_inputs = copy.deepcopy(inputs)
+        ov_model.to("AUTO")
+        self.assertTrue("AUTO" in ov_model._device)
```

Comment:
I wonder why you don't do the exact match here?

Reply:
really why? :) changed